test/parallel/test-net-server-try-ports.js

```diff
@@ -1,52 +1,42 @@
 'use strict';
-// This tests binds to one port, then attempts to start a server on that
+// This test binds to one port, then attempts to start a server on that
 // port. It should be EADDRINUSE but be able to then bind to another port.
-require('../common');
-var assert = require('assert');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
 
-var server1listening = false;
-var server2listening = false;
-var server2eaddrinuse = false;
-
-var server1 = net.Server(function(socket) {
+const server1 = net.Server(function(socket) {
   socket.destroy();
 });
 
-var server2 = net.Server(function(socket) {
+const server2 = net.Server(function(socket) {
   socket.destroy();
 });
 
-var server2errors = 0;
+let server2errors = 0;
+
 server2.on('error', function(e) {
   server2errors++;
```

Comment:
Why not get rid of this variable too and wrap the `'error'` event handler in a `common.mustCall()`?